### PR TITLE
perf(ext/websocket): optimize `op_ws_next_event`

### PR DESCRIPTION
**Pull request by littledivy**
Towards https://github.com/denoland/deno/issues/16315

### CODE REVIEW

File: ext/websocket/01_websocket.js [hunk 1]
```diff
@@ -82,6 +82,10 @@
   const _idleTimeoutDuration = Symbol("[[idleTimeout]]");
   const _idleTimeoutTimeout = Symbol("[[idleTimeoutTimeout]]");
   const _serverHandleIdleTimeout = Symbol("[[serverHandleIdleTimeout]]");
+
+  /* [event type, error code] */
```
Comment (crowlKats) [inline ext/websocket/01_websocket.js:86]: nitpick: "error code" isnt really the right description

File: ext/websocket/01_websocket.js [hunk 2]
```diff
@@ -424,18 +432,23 @@
             this.dispatchEvent(event);
             break;
           }
-          case "ping": {
+          /* ping */
+          case 3: {
             core.opAsync("op_ws_send", this[_rid], {
               kind: "pong",
             });
```
Comment (crowlKats) [inline ext/websocket/01_websocket.js:439]: we can actually delete this because tungstenite auto-responds

Comment (littledivy) [reply ext/websocket/01_websocket.js:439]: i don't want to touch funationality in this PR. maybe a follow up?

Comment (crowlKats) [reply ext/websocket/01_websocket.js:439]: sure

**Review by crowlKats — APPROVED** [review]

**Pull request merged**